WooCommerce/WCObject.cs

```diff
@@ -360,11 +360,11 @@ public WCObject(RestAPI api)
             return int.Parse(json.Substring(json.IndexOf(':') + 1, json.IndexOf('}') - json.IndexOf(':') - 1).Trim('"'));
         }
 
-        public async Task<string> GetWebhook(int id, Dictionary<string,string> parms = null)
+        public async Task<Webhook> GetWebhook(int id, Dictionary<string,string> parms = null)
         {
             string json = await API.SendHttpClientRequest("webhooks/" + id.ToString(), RequestMethod.GET, string.Empty, parms);
             json = json.Substring(json.IndexOf(":{") + 1, json.Length - json.IndexOf(":{") - 2);

```

Comment:
The below code will work for both cases.
`json = json.Substring(json.IndexOf(':') + 1, json.Length - json.IndexOf(':') - 2);`